Classes/Utility/QbankUtility.php

```diff
@@ -164,4 +165,19 @@ public static function getAutoUpdateOption(): int
 
         return $extensionConfigurationManager->getAutoUpdate();
     }
+
+    /**
+     * Returns true if request exception seems to be due to that the media is permanently deleted in QBank.
+     *
+     * @param RequestException $re
+     * @return bool
+     */
+    public static function qbankRequestExceptionStatesMediaIsDeleted(RequestException $re): bool
+    {
+        if (strpos(strtolower($re->getMessage()), 'media is permanently deleted') > 0) {
```

Comment:
Oh, this is so ugly, but I can't think of a better way either.

Reply:
yes, I now :smile: